frontend/src/routes/createLearningKit.server.ts

```diff
@@ -0,0 +1,22 @@
+import {superValidate} from "sveltekit-superforms";
+import {zod} from "sveltekit-superforms/adapters";
+import {CreateKitSchema} from "$lib/models/kit";
+//import {createLearningKit} from "$lib/api/learningKit";
+import {handleApiError} from "$lib/api/ApiError";
+import {type Actions, fail} from "@sveltejs/kit";
+
+export const load = async () => {
+    const form = await superValidate(zod(CreateKitSchema));
+    return { form };
+};
+
+export const actions = {
+    create: handleApiError(async ({ request }) => {
+        const form = await superValidate(request, zod(CreateKitSchema));
+        if (!form.valid) {
+            return fail(400, {form});
+        }
+        //const idk = await createLearningKit(form.data); //what should createlearningkit return?
```

Comment:
probably nothing, but redirect to the new kit. the response shape also only need the id I think btw in the dto for this behaviour